app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -309,19 +309,29 @@ public void setRightCardVisibility(boolean visible) {
     }
 
     @Override
-    public void setBottomCardVisibility(@UploadPage int page) {
+    public void setBottomCardVisibility(@UploadPage int page, int uploadCount) {
```

Comment:
Don't know if modifying this is the best place to do it - based off the method name I wouldn't expect it to also modify subtitle visibility.

I'd maybe create a new method in `UploadPresenter` called something like `updateSubtitleVisibility()` which should use the result of `uploadModel.getCount()` to hide/show `categories_subtitle` and `license_subtitle`. This could then be called from `updateContent()`.

Reply:
Good idea, this should be implemented in this way @zhao-gang .

Reply:
OK.